lib/image_processing/vips/color.rb

```diff
@@ -0,0 +1,686 @@
+        "grey43" => [110, 110, 110],
+        "gray42" => [107, 107, 107],
+        "grey42" => [107, 107, 107],
+        "DimGray" => [105, 105, 105],
+        "DimGrey" => [105, 105, 105],
+        "gray41" => [105, 105, 105],
+        "grey41" => [105, 105, 105],
+        "gray40" => [102, 102, 102],
+        "grey40" => [102, 102, 102],
+        "gray39" => [ 99, 99, 99],
+        "grey39" => [ 99, 99, 99],
+        "gray38" => [ 97, 97, 97],
+        "grey38" => [ 97, 97, 97],
+        "gray37" => [ 94, 94, 94],
+        "grey37" => [ 94, 94, 94],
+        "gray36" => [ 92, 92, 92],
+        "grey36" => [ 92, 92, 92],
+        "gray35" => [ 89, 89, 89],
+        "grey35" => [ 89, 89, 89],
+        "gray34" => [ 87, 87, 87],
+        "grey34" => [ 87, 87, 87],
+        "gray33" => [ 84, 84, 84],
+        "grey33" => [ 84, 84, 84],
+        "gray32" => [ 82, 82, 82],
+        "grey32" => [ 82, 82, 82],
+        "gray31" => [ 79, 79, 79],
+        "grey31" => [ 79, 79, 79],
+        "gray30" => [ 77, 77, 77],
+        "grey30" => [ 77, 77, 77],
+        "gray29" => [ 74, 74, 74],
+        "grey29" => [ 74, 74, 74],
+        "gray28" => [ 71, 71, 71],
+        "grey28" => [ 71, 71, 71],
+        "gray27" => [ 69, 69, 69],
+        "grey27" => [ 69, 69, 69],
+        "gray26" => [ 66, 66, 66],
+        "grey26" => [ 66, 66, 66],
+        "gray25" => [ 64, 64, 64],
+        "grey25" => [ 64, 64, 64],
+        "gray24" => [ 61, 61, 61],
+        "grey24" => [ 61, 61, 61],
+        "gray23" => [ 59, 59, 59],
+        "grey23" => [ 59, 59, 59],
+        "gray22" => [ 56, 56, 56],
+        "grey22" => [ 56, 56, 56],
+        "gray21" => [ 54, 54, 54],
+        "grey21" => [ 54, 54, 54],
+        "gray20" => [ 51, 51, 51],
+        "grey20" => [ 51, 51, 51],
+        "gray19" => [ 48, 48, 48],
+        "grey19" => [ 48, 48, 48],
+        "gray18" => [ 46, 46, 46],
+        "grey18" => [ 46, 46, 46],
+        "gray17" => [ 43, 43, 43],
+        "grey17" => [ 43, 43, 43],
+        "gray16" => [ 41, 41, 41],
+        "grey16" => [ 41, 41, 41],
+        "gray15" => [ 38, 38, 38],
+        "grey15" => [ 38, 38, 38],
+        "gray14" => [ 36, 36, 36],
+        "grey14" => [ 36, 36, 36],
+        "gray13" => [ 33, 33, 33],
+        "grey13" => [ 33, 33, 33],
+        "gray12" => [ 31, 31, 31],
+        "grey12" => [ 31, 31, 31],
+        "gray11" => [ 28, 28, 28],
+        "grey11" => [ 28, 28, 28],
+        "gray10" => [ 26, 26, 26],
+        "grey10" => [ 26, 26, 26],
+        "gray9" => [ 23, 23, 23],
+        "grey9" => [ 23, 23, 23],
+        "gray8" => [ 20, 20, 20],
+        "grey8" => [ 20, 20, 20],
+        "gray7" => [ 18, 18, 18],
+        "grey7" => [ 18, 18, 18],
+        "gray6" => [ 15, 15, 15],
+        "grey6" => [ 15, 15, 15],
+        "gray5" => [ 13, 13, 13],
+        "grey5" => [ 13, 13, 13],
+        "gray4" => [ 10, 10, 10],
+        "grey4" => [ 10, 10, 10],
+        "gray3" => [ 8, 8, 8],
+        "grey3" => [ 8, 8, 8],
```

Comment:
Instead of duplicating `gray` & `grey`, shall we handle the conversion `grey` => `gray` from the `Color.get` method?